packages/scm/src/browser/scm-widget.tsx

```diff
@@ -544,7 +544,9 @@ class ScmResourceGroupsContainer extends React.Component<ScmResourceGroupsContai
         }
     }
     componentDidMount() {
-        this.props.addScmListKeyListeners(this.props.id);
+        setTimeout(() => {
```

Comment:
If this is being used to reset the stack, please check it works with a timeout of `0` and use that instead of an arbitrary `300` ms.

Reply:
done

Reply:
Reverted the zero timeout back, it sometimes doesn't work

Reply:
Perhaps there is a race condition this is masking?

Reply:
fixed